pages/app/ExploreDetail.js

```diff
@@ -334,6 +341,11 @@ class ExploreDetail extends Page {
     this.props.setModalOptions(options);
   }
 
+  handleOpenInExplore() {
```

Comment:
As long as is a link to another page, should we use a Link or a Router.push? 
You can pass params:
- `<Link route="explore" params={{ layers: [dataset.id] }}/>` or
- `Router.pushRoute('explore', { layers: [dataset.id] });`

What do you think? Do you think it could work? That way you will avoid ```replaceRoute = true```